pkg/devfile/adapters/kubernetes/component/commandhandler.go

```diff
@@ -0,0 +1,130 @@
+package component
+
+import (
+	"time"
+
+	devfilev1 "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	"k8s.io/klog"
+
+	"github.com/redhat-developer/odo/pkg/component"
+	"github.com/redhat-developer/odo/pkg/devfile/adapters/common"
+	"github.com/redhat-developer/odo/pkg/libdevfile"
+	"github.com/redhat-developer/odo/pkg/log"
+	"github.com/redhat-developer/odo/pkg/remotecmd"
+	"github.com/redhat-developer/odo/pkg/util"
+)
+
+const numberOfLinesToOutputLog = 100
+const processStatusWaitTimeSeconds = 1
```

Comment:
Please use `1*time.Second` here instead of converting `int` value to time value. It's more readable and less error-prone, IMO.